src/login/logind-session.h

```diff
@@ -106,6 +106,7 @@ struct Session {
         bool stopping:1;
 
         bool was_active:1;
+        bool leaved:1;
```

Comment:
left

Reply:
thank you for your reply. But I don’t understand the meaning of left here. Can you explain it further? please。

Reply:
"leaved" is not the correct word...